src/components/utils/src/file_system_win.cc

```diff
@@ -240,24 +240,27 @@ void file_system::remove_directory_content(const std::string& directory_name) {
   do
   {
     if (FILE_ATTRIBUTE_DIRECTORY == ffd.dwFileAttributes) {
-      RemoveDirectory(ffd.cFileName);
-	} else {
-	  remove(ffd.cFileName);
-	}
+      if (strncmp(ffd.cFileName, ".", 1) != 0 &&
```

Comment:
Do windows have some constants for ``.`` and ``..``?
If not it will be great add them in our source/